### PR TITLE
Nazar/feature/business routes

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: backend/app/model/business.py [hunk 1]
```diff
@@ -16,12 +16,21 @@ class Business(Base):
...
     user = relationship("User")
     products = relationship("Product", viewonly=True)
 
+    def to_dict(self):
+        self_data = {
+            "user_id": self.user_id,
+            "name": self.name,
+            "logo": self.logo,
+            "web_site_id": self.web_site_id,
+        }
```
Comment (kipod) [inline backend/app/model/business.py:32]: If you correct user ```pydantic``` library - you newer need method like that! Plese see my comments in https://github.com/Simple2B/fish-market/pull/5

File: backend/app/model/business.py [hunk 2]
```diff
@@ -16,12 +16,21 @@ class Business(Base):
 
     id = Column(Integer, primary_key=True)
     user_id = Column(Integer, ForeignKey("users.id"))
-    name = Column(String(128))
-    logo = Column(String(128))
+    name = Column(String(128), default="")
```
Comment (SvyatoslavArtymovych) [inline backend/app/model/business.py:19]: Maybe better to allow this field be nullable?

**Review by kipod — COMMENTED** [review]
Well done!!!
Please fix my comments on review https://github.com/Simple2B/fish-market/pull/5

File: backend/app/model/business.py (hunk 2 above)
Comment (SvyatoslavArtymovych) [reply backend/app/model/business.py:19]: Instead of empty string

File: backend/app/router/business.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+
+router = APIRouter(prefix="/business", tags=["business"])
+
+
+@router.get("/")
+def get_business_cur_user(
+    db: Session = Depends(get_db), current_user: m.User = Depends(get_current_user)
+):
+    log(log.INFO, "get_business_cur_user")
```
Comment (SvyatoslavArtymovych) [inline backend/app/router/business.py:18]: log(log.INFO, "get_business_cur_user [%s]", current_user)

To know which user did this request

File: backend/app/router/business.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    business: m.Business = (
```
Comment (SvyatoslavArtymovych) [inline backend/app/router/business.py:19]: Will current_user.businesses works?

File: backend/app/model/business.py (hunk 1 above)
Comment (Nazar-kob) [reply backend/app/model/business.py]: done

File: backend/app/router/business.py (hunk 3 above)
Comment (Nazar-kob) [reply backend/app/router/business.py:18]: done

**Pull request merged**